.github/workflows/operate-ci.yml

```diff
@@ -74,6 +74,16 @@ jobs:
       test-type: Integration Tests
       runner-type: gcp-core-32-default
 
+  run-importer-tests:
```

Comment:
I'm pretty sure none of these tests were running before at all. This PR fixes the existing ones and adds some more to validate the new batching functionality